src/main/java/com/zendesk/maxwell/schema/columndef/DateColumnDef.java

```diff
@@ -12,12 +11,10 @@ public DateColumnDef(String tableName, String name, String type, int pos) {
 	}
 
 	private static SimpleDateFormat dateFormatter;
-	private static final TimeZone tz = TimeZone.getTimeZone("UTC");
 
 	protected static SimpleDateFormat getDateFormatter() {
 		if ( dateFormatter == null ) {
 			dateFormatter = new SimpleDateFormat("yyyy-MM-dd");
-			dateFormatter.setTimeZone(tz);
```

Comment:
@osheroff Do you mind handling that change when merging in? I'm a bit confused about this. All our servers always run in UTC so for our use it doesn't matter either way. I can look deeper if you'd  prefer.